src/Lesson3/HomeTask5.java

```diff
@@ -0,0 +1,14 @@
+//5. Given 2 strings, a and b. Create a string c of the form short+long+short, with the shorter string on the outside and the longer string on the inside.
+//
+//   Task: a = "hop", b = "hello"
+package Lesson3;
+
+public class HomeTask5 {
+    public static void main(String[] args){
+        String a = "hop";
+        String b = "hello";
+        if (b != a) {
+            System.out.println(a + " " + b + " " + a);
+        }
```

Comment:
1. Here you check if one string != another one but indeed you have to check length of these strings.
2. Here should be if/else